packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -292,6 +287,19 @@ export const getStyles = memoizeFunction(
               display: 'inline-block',
               marginBottom: '8px'
             },
+            ':after': {

```

Comment:
I'm good with these changes, the pseudo border is a pretty common approach for applying borders that don't affect the box model. I have two thoughts/concerns:

Can we use getFocusStyle here like on [line 125](https://github.com/OfficeDev/office-ui-fabric-react/blob/08490c6fd49de1ce064480049dbca3b5d5281e3c/packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts#L125)

And with this level of visual change, we need to over communicate for sure. Anyone that might be modifying the existing border styles (either for valid or invalid reasons) needs to know that these changes are coming.

Reply:
Where would we communicate and how?